### ISSUE TITLE
About Vulkan, sort of.

### ISSUE THREAD

**Issue by LupusEtFlos** [soulsmods/DSMapStudio#464]
I'm using an old computer with no dedicated GPU, I cannot upgrade. The tool crashes on startup, since my iGPU has no Vulkan support. I merely want to merge some files into my regulations.bin to use with the PS4, yes it's possible, since that's the only platform I can run the game on.
Is there a way or am I out of luck? Thanks in advance.

**Comment by Philiquaz** [comment]
There's been talk of making a headless version, due to the tools mapstudio has but the requirements of the rendering engine.
There's currently a project that exposes some of mapstudio's tools named dsmsportable by mountlover, and this might suffice for your usecase.

**Comment by LupusEtFlos** [comment]
Thank you kindly. I'll give that a shot, then. And hopefully start modding Elden Ring much like Bloodborne has been on PS4.

**Comment by Philiquaz** [comment]
Recently there's been some work to the renderer which is pushing requirements even further (knocking out some older cards like the nvidia 700s)
Along with that has been some thought of a backup renderer, such as tested at https://github.com/soulsmods/DSMapStudio/pull/493, which should be capable of running the map and text editors
You can test a build from here https://github.com/soulsmods/DSMapStudio/actions/runs/5231923125, though it might be buggier than a typical mapstudio release.

**Comment by kingborehaha** [comment]
Update: Newer versions of DSMapStudio have implemented a low requirements exe, which lets you edit params/text without vulkan.

**Comment by LupusEtFlos** [comment]
It's been almost one year but I'd like to firstly apologise about the delayed response, and secondly thank you guys for the help, and for giving support to lesser hardware. I'm happy to report the low requirements build works for me, even on Linux under Wine 9 like a newer issue mentioned a few months back. Not only that, but I've successfully edited a PC mod and converted it to PS4 using your tool, I had to manually grab the eboot.bin and change the game type or it would recognise itself as Demon's Souls. Now onto making mods for this game, maybe Bloodborne as well while I'm at it.

Again, thank you kindly so.
